backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -514,4 +542,29 @@ private void addReportAttachments(BacktraceReport report) {
             }
         }
     }
+
+    /**
+     * Custom callback to be executed on server response to a unique events submission request
+     *
+     * @param callback object with method which will be executed
+     */
+    public void setUniqueEventsOnServerResponse(EventsOnServerResponseEventListener callback) {
```

Comment:
woudln't it be better if we will have this methods available in the backtraceMetrics class?

Reply:
Ah sorry, will move this to metrics class too!